lisa/trace.py

```diff
@@ -397,10 +397,15 @@ def get_task_pid(self, task):
         """
         if isinstance(task, str):
             pid_list = self.get_task_by_name(task)
+
+            if not pid_list:
+                raise ValueError("trace does not have any task named \"{}\".format(task)")
```

Comment:
nit: double quotes can be used as-is in single-quoted strings, without the need to escape them (and vise versa)